app.test.js

```diff
@@ -37,5 +37,25 @@ describe('Server', () => {
       expect(response.status).toBe(404);
       expect(response.body.error).toEqual(`Could not find palette with an id of ${invalidId}`);
     })
-  })
-})
+  });
+
+  describe('POST /api/v1/projects', () => {
+
+    it('Should return a 201 status and the project id', async () => {
+      const newProject = {name: 'Test Project'};
+      const response = await request(app).post('/api/v1/projects').send(newProject);
+      const projects = await database('projects').where('id', response.body.id[0]);
+      const project = projects[0];
+      expect(response.status).toBe(201);
+      expect(project.name).toEqual(newProject.name);
+    });
+
+    it('Should return a 422 status and an error message if a name property is not included in the request body', async () => {
+      const response = await request(app).post('/api/v1/projects').send({});
+      expect(response.status).toBe(422);
+      expect(response.body.error).toEqual('Expected body format {name: <String>}. You\'re missing the required name property');
```

Comment:
My 422 message is slightly different than this one.  I will adjust mine before I push up so they are consistent.

Reply:
Oops, I should've looked at yours. I'm happy to change mine if you think yours is better.